README.md

```diff
@@ -89,6 +89,8 @@ To translate between LLVM IR and SPIR-V:
     ```
     llvm-spirv -r input.spv
     ```
+    Recommendations:
+    * `-spirv-ocl-builtins-version` - to specify target version of OCL builtins to translate to (default CL1.2)
```

Comment:
Recommendations -> Recommended options
of OCL builtins -> of OpenCL builtins

Reply:
Done